jamjar/jamjar/concerts/serializers.py

```diff
@@ -70,4 +86,19 @@ def create(self, validated_data):
 
         return concert
 
-    # def update(self, instance, validated_data):
+    def get_thumbs(self, obj):
+        """
+        Return the thumbnails from the first 3 videos in the concert (or all
+        videos if there's <= 3)
+        """
+        first_videos = obj.videos.all()[:3]
```

Comment:
yeah, that's what I was thinking, but since we don't have voting yet we'll just leave it I guess.  I think once we implement that we can do sorting things.